test/test_sax_parser.rb

```diff
@@ -254,21 +254,21 @@ def test_parse_error
     assert_equal("start_element: Results, attr: {}", result[i+=1])
     assert_equal("start_element_ns: Results, attr: {}, prefix: , uri: , ns: {}", result[i+=1])
     assert_equal("characters: \n", result[i+=1])
-    assert_equal("error: Fatal error: EndTag: '</' not found at :2.", result[i+=1])
+    assert_equal("error: Fatal error: Premature end of data in tag Results line 1 at :2.", result[i+=1])
     assert_equal("end_document", result[i+=1])
 
     refute_nil(error)
     assert_kind_of(LibXML::XML::Error, error)
-    assert_equal("Fatal error: EndTag: '</' not found at :2.", error.message)
+    assert_equal("Fatal error: Premature end of data in tag Results line 1 at :2.", error.message)
     assert_equal(LibXML::XML::Error::PARSER, error.domain)
-    assert_equal(LibXML::XML::Error::LTSLASH_REQUIRED, error.code)
+    assert_equal(LibXML::XML::Error::TAG_NOT_FINISHED, error.code)
     assert_equal(LibXML::XML::Error::FATAL, error.level)
     assert_nil(error.file)
     assert_equal(2, error.line)
-    assert_nil(error.str1)
+    refuse_nil(error.str1)
```

Comment:
I think this should have been `refute_nil` instead?

Reply:
Yes.  I've opened https://github.com/xml4r/libxml-ruby/pull/175 to fix this.  Thanks.